modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/defs/BoxesEndpointDefs.scala

```diff
@@ -170,4 +170,11 @@ final class BoxesEndpointDefs[F[_]](settings: RequestsSettings) {
         "Search among UTXO set by ergoTreeTemplateHash and tokens. " +
         "The resulted UTXOs will contain at lest one of the given tokens."
       )
+
+  def getAllUnspentOutputsByAddressDef: Endpoint[(Address, Paging, SortOrder), ApiErr, Items[UOutputInfo], Any] =
+    baseEndpointDef.get
+      .in(PathPrefix / "unspent" / "all" / "byAddress" / path[Address])
+      .in(paging(settings.maxEntitiesPerRequest))
+      .in(ordering)
+      .out(jsonBody[Items[UOutputInfo]])
```

Comment:
Let's not shrink `OutputInfo` down to `UOutputInfo` (omitting some fields). I'd suggest adding new coproduct that contains both variants (`AnyOutputInfo = OutputInfo | UOutputInfo`).